src/backend/distributed/test/run_from_same_connection.c

```diff
@@ -190,6 +190,9 @@ run_commands_on_session_level_connection_to_node(PG_FUNCTION_ARGS)
 
 /*
  * override_backend_data_gpid is a wrapper around SetBackendDataGpid().
+ * Also sets distributedCommandOriginator to true since the only caller of
+ * this method calls this function actually wants this backend to
+ * be treated as a distributed command originator with the given global pid.
  */
 Datum
 override_backend_data_gpid(PG_FUNCTION_ARGS)
```

Comment:
To reviewers; this is a UDF that we only use during tests for testing purposes, and with the recent changes made in this PR, now it needs to do `SetBackendDataDistributedCommandOriginator(true);` as well.